src/main/java/eu/dissco/exportjob/repository/S3Repository.java

```diff
@@ -0,0 +1,35 @@
+package eu.dissco.exportjob.repository;
+
+import java.io.File;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
+import software.amazon.awssdk.services.s3.S3AsyncClient;
+import software.amazon.awssdk.transfer.s3.S3TransferManager;
+
+@Repository
+@RequiredArgsConstructor
+public class S3Repository {
+
+  private final S3AsyncClient s3Client;
+  private static final String BUCKET_NAME = "dissco-data-export";
+
+  public String uploadResults(File file, UUID jobId) {
+    try (var transferManager = S3TransferManager.builder().s3Client(s3Client).build()) {
+      var upload = transferManager
+          .uploadFile(uploadFileRequest -> uploadFileRequest
+              .putObjectRequest(putObjectRequest -> putObjectRequest
+                  .bucket(BUCKET_NAME)
+                  .key(jobId.toString()))
```

Comment:
Maybe create a folder structure based on days to help searches. So that not everything is directly beneath the bucket. So prefix the name with the date (year-month-day?). Probably not necessary but maybe just to be save.